redbot/core/bot.py

```diff
@@ -568,15 +581,24 @@ async def allowed_by_whitelist_blacklist(
         # All config calls are delayed until needed in this section
         # All changes should be made keeping in mind that this is also used as a global check
 
-        guild = None
         mocked = False  # used for an accurate delayed role id expansion later.
         if not who:
             if not who_id:
                 raise TypeError("Must provide a value for either `who` or `who_id`")
             mocked = True
             who = discord.Object(id=who_id)
             if guild_id:
-                guild = discord.Object(id=guild_id)
+                deprecated_removed(
+                    "`guild_id` parameter",
+                    "3.4.8",
+                    30,
+                    "Use `guild_id` parameter instead.",
```

Comment:
should this be "use `guild` paremeter instead."?

Reply:
Yes, it should, thanks!